ci/stress_tests/stress_testing_config.yaml

```diff
@@ -86,14 +86,14 @@ file_mounts: {
 # List of shell commands to run to set up nodes.
 setup_commands:
     # Consider uncommenting these if you run into dpkg locking issues
-    # - sudo pkill -9 apt-get || true
-    # - sudo pkill -9 dpkg || true
-    # - sudo dpkg --configure -a
+    - sudo pkill -9 apt-get || true
```

Comment:
Did you run into this issue? I haven't run into it before when stress testing.

Reply:
It might be the DLAMI switch.